addons/website_forum/models/forum_post.py

```diff
@@ -161,6 +162,90 @@ class Post(models.Model):
         'Can Use Full Editor',
         compute='_compute_post_karma_rights', compute_sudo=False)
 
+    def _accepted_answer(self):
+        for answer in self.child_ids:
+            if answer.is_correct:
+                return {
+                    "text": answer.plain_content,
+                    "upvoteCount": answer.vote_count,
+                    "url": answer.website_url,
+                    "datePublished": answer.create_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    "author_name": answer.create_uid.name,
+                    "author_url": f"/forum/user/{ answer.create_uid.id }",
+                }
+        return False
+
+    def _suggested_answers(self):
+        ret = []
+        for answer in self.child_ids:
+            if not answer.is_correct:
+                ret.append({
+                    "text": answer.plain_content,
+                    "upvoteCount": answer.vote_count,
+                    "url": answer.website_url,
+                    "datePublished": answer.create_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    "author_name": answer.create_uid.name,
+                    "author_url": f"/forum/user/{ answer.create_uid.id }",
+                })
+        return ret
+
+    def microdata(self):
```

Comment:
Same: verb in function name + documentation

Also I wonder if this function should not be in the `# WEBSITE` part of this file.

Reply:
Yep, I didn't see this part :)